server/datastore/mysql/software.go

```diff
@@ -2481,6 +2512,7 @@ last_vpp_install AS (
 		%s
 	FROM
 		host_vpp_software_installs hvsi
+		%s
```

Comment:
This turns into 
```
  FROM
    host_vpp_software_installs hvsi
    INNER JOIN vpp_apps va ON hvsi.adam_id = va.adam_id 
             AND hvsi.platform = va.platform
    INNER JOIN software_titles st ON va.title_id = st.id
    INNER JOIN software s ON s.title_id = st.id
    INNER JOIN software_cve ON s.id = software_cve.software_id
```

Is this the correct method of joining to get the cves for vpp apps?